packages/documentation/src/pages/index.jsx

```diff
@@ -15,11 +15,6 @@ export default function Index({ location }) {
             If you're looking to get your environment set up, visit our{' '}
             <Link to="/getting-started">Getting started</Link> section.
           </li>
-          <li>
-            If you're looking to understand how to use our visual styles and
-            components, visit our <Link to="/visual-design">Visual design</Link>{' '}
-            section.
-          </li>
```

Comment:
I figured we didn't need a link on the homepage when we have a link in the header pointing to `Components`